src/main/java/hudson/plugins/textfinder/TextFinderPublisher.java

```diff
@@ -244,6 +243,46 @@ private static Pattern compilePattern(PrintStream logger, String regexp) {
         return pattern;
     }
 
+    /** cut off first item as it is provided with getters */
```

Comment:
this comment is no longer valid. I was using only list with 0th item as primary text finder. However, now there is `primaryTextFinder` and list for others.